snap_http/api.py

```diff
@@ -262,6 +314,13 @@ def list() -> SnapdResponse:
     return http.get("/snaps")
 
 
+def list_all() -> SnapdResponse:
+    """GETs a list of installed snaps.
+
+    This stomps on builtins.list, so please import it namespaced.
```

Comment:
Looks like this comment was copied from the above function. Since this function has a different name, it's now inaccurate.